modules/installation-configuration-parameters.adoc

```diff
@@ -364,7 +354,7 @@ ifdef::openshift-origin[]
 Either `OpenShiftSDN` or `OVNKubernetes`. The default value is `OVNKubernetes`.
 endif::openshift-origin[]
 ifndef::openshift-origin[]
-Either `OpenShiftSDN` or `OVNKubernetes`. `OpenShiftSDN` is a CNI provider for all-Linux networks. `OVNKubernetes` is a CNI provider for Linux networks and hybrid networks that contain both Linux and Windows servers. The default value is `OpenShiftSDN`.
+Either `OpenShiftSDN` or `OVNKubernetes`. The default value is `OpenShiftSDN`.
```

Comment:
FYI The default is `OVNKubernetes` with OCP 4.12.
```
$ ./openshift-install version
./openshift-install 4.12.0-0.nightly-2023-01-08-142418
built from commit 9019444f1354a63991e2bc058da9539ac21061f6
release image registry.ci.openshift.org/ocp/release@sha256:0b0a195b0360934e726b42f302a9023593234f2545ef0cc1e03b294627e42fbc
release architecture amd64
$ ./openshift-install explain installconfig.networking.networkType
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  Default: "OVNKubernetes"
  NetworkType is the type of network to install. The default value is OVNKubernetes.

$

Reply:
Hello @jianli-wei - thank you for pointing out that important change in this module. That update is not going to be made in my PR, we have a team member who is tasked with changing that in the entire documentation set. I have been asked not to make that edit and to leave it alone so it can be changed automatically.